Resources/Prototypes/Recipes/Lathes/sheet.yml

```diff
@@ -29,12 +29,12 @@
     RawQuartz: 3000
 
 - type: latheRecipe
-  id: SheetRGlass
+  id: SheetRGlass1
   result: SheetRGlass1
   completetime: 0
   materials:
     RawQuartz: 100
-    RawIron: 50
+    RawIron: 40
```

Comment:
Why are you changing these? Now the price is different than the hand-crafted one.

Reply:
No, coal is also used here. If you leave everything as it was, then the question arises: why the hell is it more expensive to melt reinforced glass than to assemble it by hand? How does it even work? Therefore, I lowered the cost.

Reply:
It will simply be unprofitable. So everyone will melt glass and steel separately, and then assemble in autolathe, well, or by hand.

Reply:
Correct me if I'm wrong, but the cost seems to be exactly the same to me:
In the ore processor:
0.5 steel + 1 glass = 50 raw, iron 15 coal + 100 raw quartz
1 reinf. glass = 50 raw iron, 15 coal, 100 raw quartz
hand crafting and the lathe:
1 reinf. glass = 0.5 steel + 1 glass
The coal cost is already used when producing the glass and steel from raw ores.
Directly producing it in the lathe or ore processor should not be profitable, but simply save you the additional crafting steps.